view/frontend/templates/js/replacejs.phtml

```diff
@@ -443,8 +472,14 @@ if (!$block->isOnPageFromWhiteList() && !$block->isMinicartEnabled()) { return;
             $(bolt_button_selector).toggle(!boltRestricted);
 
             // Show magento buttons if bolt checkout is restricted, otherwise hide it
+            var style;
+            if (boltRestricted) {
+                style = 'display:inline-block !important'
+            } else {
+                style = 'display:none !important'
+            }
             $.each(toggleCheckoutConfig.magentoButtons, function( index, button ) {
-                $(button).toggle(boltRestricted);
+                $(button).attr('style', style);
```

Comment:
If done like this we are overwriting and losing eventually existing additional inline styles.

Reply:
Thank you Daniel, good catch. I've addressed this, could you please review again?